openedx/features/badging/tabs.py

```diff
@@ -1,4 +1,9 @@
+"""
+This module is essentially a broker to `xmodule/tabs.py`, It was originally introduced
+to display My Badges in Course Experience side menu for the badging feature
```

Comment:
`... to display "My Badges" in course experience side...`

Or

`... to display my badges in course experience side...`